tests/org.jboss.tools.examples.ui.bot.test/src/org/jboss/tools/examples/ui/bot/test/integration/AbstractImportQuickstartsTest.java

```diff
@@ -437,6 +441,25 @@ protected static void deleteAllProjects() {
 	protected static void cleanupShells() {
 		WorkbenchShellHandler.getInstance().closeAllNonWorbenchShells();
 	}
+	
+	protected void mavenUpdate(Quickstart quickstart) {
```

Comment:
Eh, I just noticed, I need to change this nested try catches to ifs if possible..